functions/Copy-DbaLogin.ps1

```diff
@@ -122,6 +122,22 @@ function Copy-DbaLogin {
 
         Displays all available logins on sql2016 in a grid view, then copies all selected logins to sql2017.
 
+    .EXAMPLE
+        PS C:\> $loginSplat = @{
+                    Source               = $Sql01
```

Comment:
No tabbing on multiline examples...want to show same result you would get at the console:
 
```
PS C:\> $loginSplat = @{
>> Source = $Sql01
>> Destination = "LocalHost"
...
```